ports/mimxrt/mpconfigport.h

```diff
@@ -122,18 +124,29 @@ uint32_t trng_random_u32(void);
 #ifndef MICROPY_PY_SOCKET
 #define MICROPY_PY_SOCKET                   (1)
 #endif
-#define MICROPY_PY_WEBSOCKET                (MICROPY_PY_LWIP)
-#define MICROPY_PY_WEBREPL                  (MICROPY_PY_LWIP)
+
+#ifndef MICROPY_PY_SSL
+#define MICROPY_PY_SSL                      (0)
+#endif
+#define MICROPY_PY_WEBSOCKET                (MICROPY_PY_LWIP || MICROPY_PY_NETWORK_NINAW10)
+#define MICROPY_PY_WEBREPL                  (MICROPY_PY_LWIP || MICROPY_PY_NETWORK_NINAW10)
```

Comment:
Those are just enabled unconditionally in other ports, I think it's okay now to enable them here.

Reply:
It's not enabled yet for all boards. I could not test the MIMXRT1060 and 1064 boards. And did not test the MIMXRT1176 board yet.